File: Project 9/Java Version/ListNode.java
public class ListNode {
	private int data;
	private ListNode next;
	
	ListNode(){
		this.data = 0;
		this.next = null;	
	}
	
	ListNode(int data){
		this.data = data;
		this.next = null;	
	}

	public int getData() {
		return data;
	}

	public void setData(int data) {
		this.data = data;
	}

	public ListNode getNext() {
		return next;
	}

	public void setNext(ListNode next) {
		this.next = next;
	}	
	
}

File: Project 9/C++ Version/ListNode.cpp
#include "ListNode.h"

ListNode::ListNode() {
	this->data = 0;
	this->next = NULL;
}

ListNode::ListNode(int data) {
	this->data = data;
	this->next = NULL;
}

int ListNode::getData() {
	return data;
}

void ListNode::setData(int data) {
	this->data = data;
}

ListNode* ListNode::getNext() {
	return next;
}

void ListNode::setNext(ListNode* next) {
	this->next = next;
}
